modules/recovery-services/vault/main.bicep

```diff
@@ -366,9 +366,11 @@ type privateEndpointType = {
   @description('Optional. A list of IP configurations of the private endpoint. This will be used to map to the First Party Service endpoints.')
   ipConfigurations: {
     name: string
-    groupId: string
-    memberName: string
-    privateIpAddress: string
+    properties: {
```

Comment:
On the topic of metadata, I guess we could add 1:1 the same you added the the PE module also to the customDNSConfigs property

Reply:
Added :) .. updated readme and JSONs, and reran the pipelines above to validate